src/XenCert/StorageHandler.py

```diff
@@ -88,11 +89,12 @@ def run(self):
                 XenCertPrint("Could not write through the allocated disk space on test disk, please check the storage configuration manually. Exception: %s" % str(e))
 
 class WaitForFailover(Thread):
-    def __init__(self, session, scsiid, activePaths, noOfPaths):
+    def __init__(self, session, scsiid, activePaths, noOfPaths, checkFunc):
```

Comment:
you may need to add a default value null for this new added parameter for backward compatibility?

Reply:
Needless I think, as there exist only one place using this class and it is changed correspondingly